### ISSUE TITLE
[Feature Request][macOS] Allow to use `CFBundleSupportedPlatforms` with `InfoAdditions`

### ISSUE THREAD

**Issue by itlancer** [airsdk/Adobe-Runtime-Support#3173]
## Feature Description
AIR SDK for macOS should allow to add `CFBundleSupportedPlatforms` with `InfoAdditions`.

Right now trying to add to manifest `InfoAdditions` section for macOS:
```
<key>CFBundleSupportedPlatforms</key>
    <array>
     <string>MacOSX</string>
    </array>
```
cause packaging error:
`error 105: application.macOS.InfoAdditions contains an invalid value`

Alternatively, AIR SDK could automatically fill `CFBundleSupportedPlatforms` with `MacOSX`.

Related issues:
https://github.com/airsdk/Adobe-Runtime-Support/issues/3172
https://github.com/airsdk/Adobe-Runtime-Support/issues/2857
https://github.com/airsdk/Adobe-Runtime-Support/issues/2083

## Known Workarounds
Manually add `CFBundleSupportedPlatforms` to packaged AIR app and re-sign it.

**Comment by itlancer** [comment]
@ajwfrost 
AIR 51.0.1.1 right now fills `CFBundleSupportedPlatforms` as a String.
But it should be `Array` with one `String` element.

**Comment by itlancer** [comment]
Fixed with AIR 51.0.1.2.
Thanks!
